build_tools/github_actions/lint_llvm_commit.sh

```diff
@@ -0,0 +1,64 @@
+# Copyright 2023 The StableHLO Authors.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+print_usage() {
+  echo "Usage: $0 [-f] <path/to/llvm/version>"
```

Comment:
Could we make this `<path/to/stablehlo/root>`.

We have an expectation on the user passing in the path to `llvm_version.txt` text file, but we have an assumption on where the `WORKSPACE` file is. Given that assumption, I think that this script can only be run while in the root directory.

Why not take a root and deduce `WORKSPACE` and `llvm_version.txt` based on that. Breaking if either of those files move is reasonable.

Reply:
+1